contentcuration/contentcuration/frontend/channelEdit/components/Clipboard/index.vue

```diff
@@ -24,8 +24,9 @@
         >
           <VLayout class="container pa-0 ma-0" column>
             <ToolBar
-              class="header pa-0 ma-0 ml-1"
+              class="header pa-0 ma-0"
```

Comment:
Could you make it so that the checkboxes align? 
![image](https://user-images.githubusercontent.com/7447496/99859515-1df0f300-2b45-11eb-9f3c-8d538a206ad1.png)